avaframe/com1DFA/com1DFA.py

```diff
@@ -968,7 +968,7 @@ def DFAIterate(cfg, particles, fields, dem):
     resTypesLast = list(set(resTypes + resTypesReport))
     # derive friction type
     # turn friction model into integer
-    frictModelsList = ['samosAT', 'Coulomb']
+    frictModelsList = ['samosAT', 'Coulomb', 'Voellmy']
     frictType = frictModelsList.index(cfgGen['frictModel']) + 1
```

Comment:
This fails easily i.e. if someone writes 'coulomb' or ' Coulomb'. I suggest stripping spaces and compare case - insensitive

Reply:
Done